handlers/TileMapHandler.js

```diff
@@ -244,6 +246,44 @@ class TileMapHandler {
         }
     }
 
+    resetPlayerStartPoint() {
```

Comment:
maybe it's WIP, but in general, there already is a function resetPosition in Player.js. not sure what your plan is with it. expand/delete it?

Reply:
If I remember right, this is based on code in StartFlag (which is probably still in the main branch now) that seemed to be running every time a StartFlag is instantiated. The plan for code seemed to be to find and adopt the start point for the level, so the appropriate position values are passed to the player. I refactored that so that it should only happen once when loading the level, but if that's redundant now, this is definitely worth trimming or deleting.